ci-operator/step-registry/sandboxed-containers-operator/install-trustee-operator/sandboxed-containers-operator-install-trustee-operator-commands.sh

```
@@ -0,0 +1,702 @@
+      echo ">>> INFO: Trustee KBS is responding (resource not found is expected for test resource)"
+      echo ">>> This confirms connectivity is working"
+    else
+      echo ">>> WARNING: Failed to connect to Trustee KBS"
+      echo ">>> This may be expected if resources haven't been populated yet"
+      cat /tmp/kbs-test-output.txt || true
+    fi
+  fi
+
+  # Test if we can reach the health/version endpoint
+  echo ">>> Testing Trustee KBS version endpoint"
+  if oc exec ${kbs_client_pod} -n ${kbs_client_namespace} -- \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)